store-implementation/proxy-store/src/test/java/uk/gov/gchq/gaffer/proxystore/integration/ProxyStoreBasicIT.java

```diff
@@ -216,13 +218,13 @@ public void shouldCatchAndThrowUsefulErrorMessages() throws Exception {
     }
 
     @Test
-    public void shouldHaveAllOfDelegateStoreTraitsApartFromVisibility() {
+    public void shouldHaveAllOfDelegateStoreTraitsApartFromVisibility() throws OperationException {
         // Given
         final Set<StoreTrait> expectedTraits = new HashSet<>(MapStore.TRAITS);
         expectedTraits.remove(StoreTrait.VISIBILITY);
 
         // When
-        final Set<StoreTrait> storeTraits = graph.getStoreTraits();
+        final Set<StoreTrait> storeTraits = graph.execute(new GetTraits.Builder().currentTraits(false).build(), new Context());
```

Comment:
If the Operation is meant to replace the function, perhaps `currentTraits` should default to false?

Reply:
It could and probably should have done from the start - if it was actually intended to do that. I don't think we should change it in this PR. Might be something to consider if we remove the REST endpoints - I've proposed this in #2913.